lib/modules/manager/gleam/readme.md

```diff
@@ -15,12 +15,13 @@ The `gleam` manager, however, uses `gleam` itself to keep `manifest.toml` up-to-
 
 Renovate's `"auto"` strategy defaults to `"widen"` and works like this for `gleam`:
 
+<!-- prettier-ignore -->
```

Comment:
I'm okay with this as hack-job.

But we could try setting `4 spaces` as `tabwidth` for Prettier, in a new PR. That way we can see if that works better for us, out of the box.

Reply:
I would be more than happy to create a PR using `"tabWidth": 4`, however this will make prettier reformat the entire repo and start enforcing a non-standard tab width going forward. It is also possible such a change could unintentionally break other, existing docs as well.

Lastly, I suspect setting `"tabWidth": 4` might be more of a work around for this specific prettier & mkdocs incompatibility rather than being the objectively correct thing to do.

With all of that said, may I propose we leave the `prettier-ignore` in-place and add a comment/todo referencing this PR as a form of documentation for this specific format exception? If that is undesirable, I could instead explore re-writing that docs section to forgo any embedded lists.

As an outside contributor, I'm unsure which direction would be most preferable from the project's perspective. Waiting on insight/guidance from @rarkins and @HonkingGoose

Reply:
we can use the editor config to limit it to markdown files

Reply:
That is a good point @viceice. However, even with only limiting the changes to markdown files, I would be concerned with the risk of breaking other existing docs. The rendering issue here was subtle, caught me off guard, and snuck through a code review, so the concern would be that we wouldn't be able to notice/catch other new rendering issues after the formatting change.

I very well could be overthinking it though! Let me know if the consensus is we should explore this path - I'm happy to put together a new PR.

I suppose if Renovate is exploring a tabWidth change in prettier (even for just markdown files), it may be relevant to mention this ongoing discussion (https://github.com/prettier/prettier/issues/7475) regarding the use of tab chars instead of spaces for accessibility reasons. Personally, I like spaces, but have no other opinion on the subject.

Reply:
OK, PR #30608 explores setting markdown `tabWidth: 4`.

I'll need help with that one, lots of changed files. From what I can see, it mostly changed what you would expect. However, it seems to do some odd things with root-level unordered lists.

Specifically, changing:

```
- List Item A
- List Item B
```

into:

```
-   List Item A
-   List Item B
```
(note the 3 space chars after the hyphen char)

mkdocs seems to render this as you would expect, even though it looks odd. Perhaps this is a readability choice made by prettier?

For embedded unordered lists, such as the `gleam` manager example, it does the correct thing:

```
1. List Item A
    - Example A
```

(note the leading 4 space chars before the hyphen char, followed by a single space char).